core/store/src/columns.rs

```diff
@@ -237,6 +235,11 @@ pub enum DBCol {
     /// - *Rows*: BlockShardId (BlockHash || ShardId) - 40 bytes
     /// - *Column type*: StateChangesForSplitStates
     StateChangesForSplitStates,
+    /// Transaction or receipt outcome, by outcome ID (transaction or receipt hash) and block
+    /// hash. Multiple outcomes may be stored for the same outcome ID in case of forks.
+    /// *Rows*: OutcomeId (CryptoHash) || BlockHash (CryptoHash)
+    /// *Column type*: ExecutionOutcomeWithProof
+    TransactionResultForBlock,
```

Comment:
Should be mark this as `is_insert_only` further down?

Reply:
Yes! Good catch :)